logic/tool/TypeResolver.java

```diff
@@ -186,51 +236,51 @@ private TypeVariable register(ThingVariable var) {
             // have resolved its valueType, so we execute convertValue first.
             var.value().forEach(constraint -> registerValue(resolver, constraint));
             var.isa().ifPresent(constraint -> registerIsa(resolver, constraint));
-            var.is().forEach(constraint -> registerIs(resolver, constraint));
+            var.is().forEach(constraint -> registerIsThing(resolver, constraint));
             var.has().forEach(constraint -> registerHas(resolver, constraint));
             var.relation().forEach(constraint -> registerRelation(resolver, constraint));
             var.iid().ifPresent(constraint -> registerIID(resolver, constraint));
             return resolver;
         }
 
-        private void registerIID(TypeVariable owner, IIDConstraint iidConstraint) {
+        private void registerIID(TypeVariable resolver, IIDConstraint iidConstraint) {
```

Comment:
Decided to make `resolver` used more consistently.

Reply:
great :)